tests/api.test.js

```diff
@@ -35,10 +35,10 @@ describe('api', () => {
     });
 
     it('should generate transfer bxml', () => {
-        var bxmlString = ApiController.generateTransferBxml("test-string");
+        var bxmlString = ApiController.generateTransferBxml("test-string", "c-93d6f3c0-be584596-0b74-4fa2-8015-d8ede84bd1a4");
         bxmlString = bxmlString.replace(/\s/g, "");
 
-        var expectedString = '<?xmlversion="1.0"encoding="UTF-8"?><Response><Transfer><SipUriuui="test-string;encoding=jwt">sip:sipx.webrtc.bandwidth.com:5060</SipUri></Transfer></Response>';
+        var expectedString = '<?xmlversion="1.0"encoding="UTF-8"?><Response><Transfer><SipUriuui="93d6f3c0be5845960b744fa28015d8ede84bd1a4;encoding=base64,test-string;encoding=jwt">sip:sipx.webrtc.bandwidth.com:5060</SipUri></Transfer></Response>';
```

Comment:
Perhaps add another test where a custom SIP URI is supplied

Reply:
Good suggestion - added!